### ISSUE TITLE
Sumbit50 not installing error code

### ISSUE THREAD

**Issue by ranielbabu** [cs50/submit50#244]
#243 My system does not meet the requirements to download Linux subsystem for windows so is there any other way to install submit50 on to my desktop?
![](https://user-images.githubusercontent.com/50634513/90336065-03f1c280-dfe2-11ea-96da-f5f81a2c5bb0.png)

**Comment by curiouskiwi** [comment]
No, it is not compatible in Windows unless installed under WSL. 
If you are taking CS50X, there is no need to install submit50 since it is preinstalled in the CS50 IDE that we strongly recommend all students use.

**Comment by ranielbabu** [comment]
> No, it is not compatible in Windows unless installed under WSL.
> If you are taking CS50X, there is no need to install submit50 since it is preinstalled in the CS50 IDE that we strongly recommend all students use.

but i am doing cs50 web development with python and JavaScript by Brian. I have already completed cs50 X, when i go to cs50 ide it says CS50 IDE is down for some users.

**Comment by curiouskiwi** [comment]
You can either upload your files to the IDE to submit them, or learn to use git and submit them that way, per the instructions.  In the future, it's best to ask questions like this in one of our [communities](https://cs50.harvard.edu/web/2020/communities/).  This isn't a submit50 issue.
